controls/V-72253.rb

```diff
@@ -57,20 +57,12 @@
 
 The SSH service must be restarted for changes to take effect."
 
-  # Must be either or both of these ciphers
-  # @todo - replace with be_in when available
-  describe.one do
-    describe sshd_config do
-      its('MACs') { should match /^hmac-sha2-256,hmac-sha2-512$/ }
-    end
-    describe sshd_config do
-      its('MACs') { should match /^hmac-sha2-256$/ }
-    end
-    describe sshd_config do
-      its('MACs') { should match /^hmac-sha2-512$/ }
-    end
-    describe sshd_config do
-      its('MACs') { should match /^hmac-sha2-512,hmac-sha2-256$/ }
+  @macs = inspec.sshd_config.params("macs")
```

Comment:
Previous comments from Trevor...

"I'm not 100% sure that sha-512 works in FIPS mode (and I'm 80% sure that it doesn't in all versions of RHEL). This should be checked before it's pushed.

We're going to have to toggle some of these based on FIPS being enabled on the target system."

Reply:
Thanks @jburns12 . My comment still holds. Are these rules supposed to be for EL7.0+ or just the latest EL7, or what? My preference would be that they are accurate for EL7.0+ if possible. If not, they need to fail on anything that they don't support.

Reply:
@trevor-vaughan I chose these two hashes because they are what's listed in the check of the control. After doing some research, it looks like hmac-sha1 could be considered FIPS compliant as well, so I'm not sure whether or not this should be included.

Reply:
@jburns12 I believe you about the check. I'd also like to ask if anyone actually *tried* those values with success on 7.0. Presently, it looks like the validation on the NIST site doesn't go above `sha-384` and SSH **will crash** if it's using an invalid cipher set.

You are correct that `sha1` is allowed. It's generally discouraged except for backwards compatibility and should not be used in production if possible.

Reply:
@trevor-vaughan Sorry...I didn't originally get what you were saying but I do now. It sounds like the `sha-256` flavor should work but `sha-512` might crash as you're saying. I'll try to get some time to work on this next week and see if i can test on a spun up system.

Reply:
I just validated that sha-512 does not cause an issue on the earliest version of EL7. I am going to add in sha1 support since it is valid per NIST and push things through.